src/DynamoCore/Linting/LinterExtensionDescriptor.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Linting
+{
+    public class LinterExtensionDescriptor
```

Comment:
this looks pretty similar to: https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Extensions/ExtensionData.cs#L6

Reply:
@BogdanZavu @SHKnudsen 
1. does this class need to exist? See the very similarExtensionData class, could this just be a tuple instead? Do we expect this to grow
2. Does this class really need to be public?

Reply:
So ExtensionData has a few more things in it and I feel that using it instead of LinterDescriptor will make things more confusing. I guess the most valuable thing of LinterDescriptor is the actually the name. I would expect that there are many similar or almost similar simple structs across dynamo. I think we can have a tuple here but I would prefer the named structure instead. Most likely it will not grow or if it will grow it will grow very little.
I think it can be internal.

Reply:
@BogdanZavu - sounds good to me if thats possible.

Reply:
I forgot about the view, we need it there (in the other [PR](https://github.com/DynamoDS/Dynamo/pull/11634/files) ) where the active linter is being set from the UI.

Reply:
ah